monarch_ingest/hgnc/gene2publication.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+
+from biolink_model_pydantic.model import (
+    Gene,
+    NamedThingToInformationContentEntityAssociation,
+    Predicate,
+    Publication,
+)
+from koza.cli_runner import koza_app
+
+source_name = "hgnc_gene_to_publication"
+
+row = koza_app.get_row(source_name)
+
+if not row["pubmed_id"]:
+    koza_app.next_row()
+
+
+gene = Gene(id=row["hgnc_id"])
```

Comment:
I was thinking on this one that we would probably consider this our primary source for these gene nodes, so we would want to populate as much as we can (symbol, name, collect all of the alias & previous names fields and combine with synonyms, and maybe some xrefs (ensembl, ncbi at least) - what do you think @kshefchek?  or will we get all of that from the ontology side?

Reply:
I agree with this unless @matentzn or @cmungall say otherwise. Perhaps we should name this source gene info instead?

Reply:
That makes sense to me, that this one is primarily for gene_information (and also happens to make publication associations)